documents/ms.tex

```diff
@@ -446,6 +461,25 @@ \subsubsection{Non-period Hyperparameters}
 works well may differ among datasets (see \ref{sec:} and 
 \ref{sec:} for two different examples of hyperparameter priors).
 
+\vrcomment{VR: in my opinion, this is the weakest part of the paper. I don't agree that there are no \emph{a priori} reasons to believe that any of the hyperparameters should take values in any specific range, and the phrase `somewhat subjective art' seems likely to set alarm bells ringing for referees/readers. Going through the hyperparameters in turn, the following things come to mind.
+
+\begin{itemize}
+\item Amplitude, $A$: their is a closed form solution for the maximum-likelihood value of $A$ (solve ${\partial _A}\mathcal{L} = 0$). Steve Roberts alluded to this in his 2012 tutorial paper and I know I've worked through the mathematics myself -- offhand I can't remember the solution, but it's a fairly straightforward looking thing involving the precision matrix like $A\sim{{\mathbf{y}}^{\textrm{T}}}{{\mathbf{K}}^{ - 1}}{\mathbf{y}}$ with some scaling constant. I'll see if I can dig up my workings and/or find a reference. I think this means that if you're anyway going to use a flat prior on $A$, your MAP solution will end up corresponding to the ML solution. In which case you could speed things up a good deal by just setting $A$ to its optimal value \emph{ab initio}, and not bothering to explore the full posterior PDF. More importantly though, if for some reason the MAP value you use doesn't correspond more or less to the ML value (e.g.\ if something went wrong with the MCMC convergence) -- in other words, if you end up using a non-optimal value for $A$ -- I've found you can sometimes get rather bizarre behaviour. For example, when I fit data with an obvious period, sometimes I end up recovering a very wrong period if $A$ is also very wrong e.g.\ much too large. Such problems can be avoided quite straightforwardly by constraining $A$ to sensible values.
```

Comment:
This must be the ML value of A _conditioned on_ the other parameters though. There's no reason why the ML value of A for some values of the other hyperparameters will be the MAP of the full posterior! I agree that you could use this to remove a parameter from the inference but I'm not convinced that that would make a huge difference.